app/models/calculation_result.rb

```diff
@@ -91,6 +96,10 @@ def pensioner_disregard_applied?
     api_response.dig(:result_summary, :capital, :pensioner_capital_disregard).positive?
   end
 
+  def pensioner_disregard_value
```

Comment:
You only use this in one place, and to check whether `pensioned_disregard_applied` is positive. So I think you're better off, instead of creating this method, updating the now incorrect `pensioner_disregard_applied?` - if it checked `:pensioner_disregard_applied` rather than `:pensioner_capital_disregard` it would be more accurate, and save us from having the term "pensioner_disregard_applied" refer to 2 different things.